client/modules/IDE/pages/IDEViewMobile.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Link } from 'react-router';
+
+import { prop, remSize } from '../../../theme';
+
+const background = prop('Button.default.background');
+const textColor = prop('primaryTextColor');
+
+const Header = styled.div`
+  width: 100%;
+  background-color: ${background} !important;
```

Comment:
do you need to use `!important` here? i'd prefer not to use it ever unless absolutely necessary (e.g. overriding styles from another library).

Reply:
good point, will change this